src/app/events/components/evaluation/evaluation-table/evaluation-table.component.ts

```diff
@@ -64,6 +75,18 @@ export class EvaluationTableComponent {
   subscriptionDetailChange = output<EvaluationSubscriptionDetail>();
   gradingScale = input.required<GradingScale>();
 
+  eventId = toSignal<number | null>(
+    toObservable(this.state.event).pipe(map((event) => event?.id ?? null)),
+    { initialValue: null },
+  );
+
+  hasOpenEvaluations = toSignal(
+    toObservable(this.entries).pipe(
+      map((entries) => entries.some((entry) => entry.evaluationRequired)),
+    ),
+    { initialValue: false },
+  );
```

Comment:
@llorentelemmc für ein andermal: du konvertiest ein Signal in ein Observable, verwendest dann den `map` Operator und konvertierst das Resultat wieder zurück in ein Signal...

Was du eigentlich machen willst wäre ein `computed`:

```ts
hasOpenEvaluations = computed(() => this.entries().some((entry) => entry.evaluationRequired));
```